src/loan/loan.controller.ts

```diff
@@ -17,10 +17,11 @@ interface JWTRequest extends Request {
 export class LoanController {
     constructor(private readonly loanService: LoanService) {}
     @ApiBearerAuth()
-    @UseGuards(AuthGuard('jwt'))
+    @ApiOperation({ summary: 'Create a loan application' })
+    @ApiResponse({ status: 201, description: 'Loan created successfully' })
     @Post()
-    create(@Body() createLoanDto: CreateLoanDto, @Request() req:JWTRequest) {
-        const userId = req.user.userId;
+    create(@Body() createLoanDto: CreateLoanDto, @Request() req) {
+        const userId = req.user.id;
```

Comment:
consider the old code: refer to jwt.strategy.ts validate payload, it is userId not id.